client/consensus/src/consensus_orchestrator.rs

```diff
@@ -436,6 +436,7 @@ where
         }
     }
 
+    // TODO: Where to put these info in the refactor?
     fn pre_digest_data(&self, slot: Slot, claim: &Self::Claim) -> Vec<sp_runtime::DigestItem> {
         vec![
```

Comment:
You can make this function call the other one:
```suggestion
    // TODO: Where to put these info in the refactor?
    fn pre_digest_data(&self, slot: Slot, claim: &Self::Claim) -> Vec<sp_runtime::DigestItem> {
        crate::collators::pre_digest_data(slot, claim)
```

Reply:
Yeah, in that case we will need to modify this function again when we finally deprecate the old params.

Reply:
As the old parameters were removed, now there's no need to modify this one.